tests/test_study.py

```diff
@@ -29,10 +29,10 @@
 
     CallbackFuncType = Callable[[optuna.study.Study, optuna.trial.FrozenTrial], None]
 
-# TODO(ytsmiling) Add tests for multi-worker settings.
```

Comment:
Just for the record, the TODO was addressed by https://github.com/optuna/optuna/blob/master/tests/storages_tests/rdb_tests/test_with_server.py.